x-pack/legacy/plugins/apm/server/lib/settings/cm/delete_configuration.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { Setup } from '../../helpers/setup_request';
+
+export async function deleteConfiguration({
+  configurationId,
+  setup
+}: {
+  configurationId: string;
+  setup: Setup;
+}) {
+  const { client, config } = setup;
+
+  const params = {
+    refresh: true,
```

Comment:
Haven't discussed it, but i can change it to `'wait_for'` so as to be unobtrusive for the rest of cluster. It adds an almost unnoticeable delay to delete, but i think it's actually OK, ux-wise since there's more gravity to the delete operation.